tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -18,38 +19,38 @@
 
     <!-- Integration: AWSSDK.Core -->
     <!--    Assembly: AWSSDK.Core -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Core/3.7.400.2 -->
-    <PackageReference Include="AWSSDK.Core" Version="3.7.400.2" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Core/3.7.400.6 -->
+    <PackageReference Include="AWSSDK.Core" Version="3.7.400.6" />
 
     <!-- Integration: AWSSDK.DynamoDBv2 -->
     <!--    Assembly: AWSSDK.DynamoDBv2 -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.DynamoDBv2/3.7.400.2 -->
-    <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.400.2" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.DynamoDBv2/3.7.400.6 -->
+    <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.400.6" />
 
     <!-- Integration: AWSSDK.Kinesis -->
     <!--    Assembly: AWSSDK.Kinesis -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Kinesis/3.7.401 -->
-    <PackageReference Include="AWSSDK.Kinesis" Version="3.7.401" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Kinesis/3.7.401.4 -->
+    <PackageReference Include="AWSSDK.Kinesis" Version="3.7.401.4" />
 
     <!-- Integration: AWSSDK.SimpleNotificationService -->
     <!--    Assembly: AWSSDK.SimpleNotificationService -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SimpleNotificationService/3.7.400.2 -->
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.400.2" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SimpleNotificationService/3.7.400.6 -->
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.400.6" />
 
     <!-- Integration: AWSSDK.SQS -->
     <!--    Assembly: AWSSDK.SQS -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SQS/3.7.400.2 -->
-    <PackageReference Include="AWSSDK.SQS" Version="3.7.400.2" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SQS/3.7.400.6 -->
+    <PackageReference Include="AWSSDK.SQS" Version="3.7.400.6" />
 
     <!-- Integration: Azure.Messaging.ServiceBus -->
     <!--    Assembly: Azure.Messaging.ServiceBus -->
-    <!-- Latest package https://www.nuget.org/packages/Azure.Messaging.ServiceBus/7.18.1 -->
+    <!-- ⚠POTENTIALLY UNSUPPORTED⚠ Latest package https://www.nuget.org/packages/Azure.Messaging.ServiceBus/7.18.1 -->
```

Comment:
I'll make a task of testing this out locally for  `v7.18.1` (done it before and I think Zach made steps on how to do it), but it isn't a priority ATM.

Reply:
Another possibility here (probably harder work, just food for thought)
- Set up an app that uses Service Bus that deploys to Azure
- Instrument with the latest dev version of the AAS extension
- Set up dependabot for servicebus on the new app so we can auto merge it
- Set up slack alerts in DD to warn if anything's changed on that test app